extensions/oidc/runtime/src/main/java/io/quarkus/oidc/Tenant.java

```diff
@@ -11,6 +11,7 @@
 
 /**
  * Annotation which can be used to associate OIDC tenant configurations with Jakarta REST resources and resource methods.
+ * It is also possible to use the annotation to associate OIDC tenant configuration with WebSockets Next endpoints.
```

Comment:
IMHO, rather than listing different transports here (how would WS Next align with the `TenantIdentityProvider` mentioned just below), and what about GRPC, it would make sense to simplify this JavaDocs and say something like `Annotation which can be used to associate OIDC tenant configurations with the endpoint methods.` with the docs providing more specific details related to JAX-RS, WS Next, etc

Reply:
It's not supported for gRPC (as many things, we concentrate mostly on Jakarta REST and WS Next). I will use your proposal as Quarkus docs explicitly mentions Jakarta REST and WS Next. We also validate that they are present when `@Tenant` instance is detected, so that's the best (fo reasonable measures) we can do. I'll drop it.